src/leveldbwrapper.cpp

```diff
@@ -92,3 +92,8 @@ bool CLevelDBWrapper::WriteBatch(CLevelDBBatch& batch, bool fSync) throw(leveldb
 
 // Taken from future release of DBWrapper
 const std::string CLevelDBWrapper::OBFUSCATE_KEY_KEY("\000obfuscate_key", 14);
+
+CLevelDBIterator::~CLevelDBIterator() { delete piter; }
+bool CLevelDBIterator::Valid() { return piter->Valid(); }
+void CLevelDBIterator::SeekToFirst() { piter->SeekToFirst(); }
+void CLevelDBIterator::Next() { piter->Next(); }
```

Comment:
Why no SeekToLast() and Prev() as in original commit?

Reply:
These are removed in a later commit. I just removed them here to avoid having to cherry-pick them away later.